nextcloudappstore/core/templates/user/delete-account.html

```diff
@@ -0,0 +1,35 @@
+{% extends "user/base.html" %}
+{% load i18n %}
+{% load account socialaccount css_class %}
+
+{% block head-title %}{% trans "Delete account" %} - {% endblock %}
+
+{% block account-content %}
+    <h1>{% trans "Delete account" %}</h1>
+    <section>
+        <form method="POST" action="{% url 'account-deletion' %}">
+            {% csrf_token %}
+            <p class="alert alert-danger">{% trans 'Deleting your account will delete all your apps and releases as well. To delete your account, insert your e-mail address into the input field and click the "Delete account" button:' %}</p>
+            {% for field in form %}
```

Comment:
I'm on mobile so it's cumbersome for me to check, but is it possible to include `user/form-fields.html` to replace most of the contents of this `<form>`?